pkg/trace/writer/stats.go

```diff
@@ -41,6 +42,8 @@ type StatsWriter struct {
 	senders  []*sender
 	stop     chan struct{}
 	stats    *info.StatsWriterInfo
+
+	easylog *logutil.ThrottledLogger
```

Comment:
Is there a reason these are named `easylog`? I don't have a real issue with it, I just find it a bit confusing. The `easy` seems like it's trying to imply something, but I don't see it explained anywhere.